include/ddc/kernels/splines/splines_linear_solver_sparse.hpp

```diff
@@ -172,30 +178,30 @@ class Matrix_Sparse : public Matrix
         m_matrix_sparse = matrix_sparse_type::create(gko_exec, gko::dim<2>(mat_size, mat_size));
     }
 
-    virtual double get_element([[maybe_unused]] int i, [[maybe_unused]] int j) const override
+    virtual double get_element([[maybe_unused]] std::size_t i, [[maybe_unused]] std::size_t j)
+            const override
     {
-        throw std::runtime_error("MatrixSparse::get_element() is not implemented because no API is "
-                                 "provided by Ginkgo");
+        throw std::runtime_error(
+                "SplinesLinearProblemSparse::get_element() is not implemented because no API is "
+                "provided by Ginkgo");
```

Comment:
```suggestion
        return m_matrix_dense->at(i, j);
```

Would this work ?

Reply:
Yes, now that lifecycle is clearly defined it makes sense. I propose commit 6033f08

Reply:
Yes good to me